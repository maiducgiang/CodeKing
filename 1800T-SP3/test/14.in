17
12
25865 61484
55558 87565
87704 -52536
1396 23931
87954 18215
48868 7232
96892 -52254
46743 81717
18603 -78919
77934 -23879
36902 -22569
38852 55091
32
33150 -33572
79243 -92285
88334 35890
65144 96238
22781 -28243
96725 23225
61437 -11063
16518 -23174
49463 61107
99191 83052
7754 -5180
22651 1054
45390 -47222
97904 66413
65615 -42166
18169 -94973
39475 -63639
33418 45876
68171 -56313
16723 -35952
64919 14556
71965 29974
32234 -32306
2432 -43245
93269 33456
48858 27359
6885 3408
1217 42134
31358 17143
2570 -77429
63515 -37303
48342 -54203
87
72699 60993
97012 44036
10251 42218
84907 49057
81846 59879
77867 81198
3859 47444
4994 -17246
11294 -97900
30893 -83152
22752 45726
69584 12872
60814 -55553
93845 -46558
66251 56099
14160 -18031
51647 -53566
41749 25294
80579 -91118
68170 -50425
50620 -42604
88386 30096
26241 21172
35193 22874
56573 -331
63697 -19648
56538 -69987
32381 -72799
5599 -2745
81834 -1501
25907 47082
84515 -6311
49948 -54475
17774 1513
60907 88167
73093 -401
37266 12764
15971 3004
39937 85278
2728 -94941
88544 81420
84952 -3535
56218 -56585
9728 57301
53241 -36153
39178 31011
18286 20037
56865 -8509
19643 -55442
42272 -23109
895 48383
70608 29975
72918 -59091
35171 -96955
72936 -52234
31779 -12155
51491 -89371
4105 -1652
15506 32022
74562 -93778
92188 51635
63305 -51547
92043 -64769
47274 47977
48850 -65667
35207 -7167
79426 60575
61645 58514
16206 85892
12547 62392
30608 -53524
27652 -35140
1009 53517
99062 -78744
77582 -55408
66660 -61522
36570 53297
69813 -49330
61510 -58681
46907 -11748
30273 16140
54563 -15560
95888 -71020
45875 -60941
83330 13804
59297 -1101
67759 51221
53
6873 -38792
61461 -73819
33716 -34112
16931 -67740
22528 -88635
41307 96978
27480 72524
91290 53632
78665 -46025
51426 -41336
69004 57454
76250 36644
25120 -10163
99585 75640
85063 -45560
95792 62213
65497 22828
62546 -811
5196 -73237
68071 59199
51624 -65635
55616 -21834
61556 -49645
55254 -39781
45853 41682
51797 -95104
46717 74788
44069 75221
86517 14002
33357 -96338
35761 82468
27209 89748
74640 -34786
95738 67442
23532 -33651
33320 -61393
55466 72614
73230 65089
29320 -14831
82276 70789
23863 71692
14086 46441
6166 -99835
82006 -27067
81714 68556
90279 -47779
69986 -46716
81862 98048
7831 6814
23906 -70035
93765 -98325
53336 -81339
65971 -59280
25
81097 -2631
32990 -82571
70341 61510
81565 -83368
97152 24306
71226 21138
84090 89804
71626 -59964
24378 -42392
85556 -21448
10431 84689
99257 -63008
47898 80703
86369 17473
58695 10179
86108 12507
65017 -84031
7632 -24445
46113 22896
61740 -37841
33617 -90050
80292 -34045
31990 -57870
59265 -87883
58892 47906
35
81845 -4221
85975 -46716
70420 98093
58324 -95539
98402 -14135
71409 -67476
76609 -60273
76659 -17026
63874 -21940
67638 19652
37346 81200
85913 -86994
33928 55158
72546 -27999
96905 54089
78562 -49960
67039 -31135
76471 -4021
56775 -73241
46007 35735
50004 -48185
77831 15892
80661 32276
27405 53624
39860 -95696
33471 -87988
50312 80252
97963 -87388
85912 -72036
24054 -43653
50213 97645
65739 44978
28056 95888
82145 -61741
25890 24549
97
47830 47852
23591 17805
10919 -14627
81117 24444
14715 45469
71882 14392
55418 44813
4327 -11971
76839 89062
66120 38120
18254 91123
89936 -18628
63586 -61486
23678 75673
357 16500
31128 99846
27842 39906
46235 91287
95928 57736
1000 73532
12127 15929
65104 -62749
48181 77931
12113 41611
5696 -93872
30805 -48157
72743 -52895
69170 27105
4621 51298
295 -6762
44010 76454
72123 40280
98072 69117
74870 -12468
61696 62479
19130 -66585
5214 -89605
50098 65199
49432 16630
61324 -49851
47621 3756
17979 87916
78188 -5013
82234 50174
48103 91486
40969 -2942
50313 -59928
59334 -86987
26059 -25131
48347 -34818
28504 -25667
40738 47342
81159 -41632
22541 -65347
31737 43180
96214 -42077
65741 -93550
5889 73578
17898 92734
62167 79750
3510 -20382
98444 49906
99963 49151
93076 -73854
7567 46896
2907 -99573
43706 -61833
43726 -37746
39506 92184
88371 43890
91930 -93469
80095 -43745
98535 -86627
58300 58887
56432 -885
88900 23995
15735 23210
13726 92197
88020 94391
25774 78847
54935 -73521
93412 26989
83374 26196
95744 -41532
56389 64585
78025 -10156
35124 -30930
58126 -41097
38102 37680
14039 62048
69152 95982
73714 88720
7974 36633
49287 31981
27896 56653
63235 -75629
20876 -70096
43
91435 95307
79749 72730
16500 -42212
74651 40072
99962 58186
71838 -64184
66153 -33621
91286 -85217
80008 -67451
86239 -40062
21264 89518
42591 -23113
94375 32664
40936 -12442
73497 12695
79360 -27302
6487 -59924
79327 87157
70670 -97552
88542 99924
38502 -28747
83641 94834
44790 50626
11284 14768
29026 12153
96922 88757
97875 -37868
98394 90977
77384 26861
19722 -82646
28897 -74908
93919 25579
58843 4409
49631 84760
24538 -75918
9216 13189
57988 87906
49801 -79502
24923 -21786
60152 76651
92419 -73059
99775 40673
60292 78023
24
61728 -37734
11103 -6801
20732 4404
40868 -86253
66605 2574
8084 -34679
29649 74312
7539 -30945
91651 54642
92551 -56148
7590 15779
81713 18656
51046 77622
51649 -55600
67710 -75778
62399 -26033
27084 -54388
44287 -29752
19209 60654
14121 60090
54487 -39561
88014 88107
4049 -1473
26169 -57590
35
93137 -36820
31755 72647
77571 43836
35518 -47758
53486 50632
78144 -63467
26401 83056
25848 18583
36562 -82978
87923 -93844
33305 25642
19184 -82548
6678 -98099
95320 -87640
2772 -31087
88008 10210
66446 97991
39521 10359
55413 4591
10529 -18579
76242 62313
13451 12703
52046 -29661
91330 18932
30985 -19067
24562 -47819
7314 7723
30979 21678
957 33323
9350 -75704
10023 16631
64963 -53278
34672 72866
73142 -99486
59921 12989
78
55235 -61754
62160 82821
2136 66898
34203 -51752
81793 -98451
81410 152
44421 91303
88949 -87199
43607 9167
89525 35406
68011 59045
94793 -36727
78098 -39659
34146 22999
22877 -21640
25520 -46488
90431 23385
85331 -26058
94994 -38965
16107 -84514
6788 -40772
1279 -52680
55433 39031
65518 36493
23666 38374
34188 46043
89430 -88405
60118 -2153
88049 -85202
73306 15013
68756 76321
24040 16429
80072 -79549
62934 62696
8196 -77087
68929 -76933
14046 61635
94844 74734
30386 -13372
16580 74300
61630 -59050
65387 -94374
80927 -16700
31930 -58980
62777 -55634
24649 28574
57239 -73278
39620 28722
36996 17708
60666 83084
40842 -91533
5222 76738
28922 99321
65787 -18549
66629 19361
25630 31829
37622 25406
65894 -49358
86733 -76806
50107 -13792
51556 -71505
29131 56712
78749 53675
97530 51887
82369 -2734
25628 7011
85969 92750
83644 -56798
22060 -35149
4510 93115
75693 -43932
99328 -10269
83552 13134
95550 -67131
83865 66549
35601 -28790
29044 -76828
24833 66126
61
90256 -98657
83103 32040
33115 48508
79565 -78122
91507 -22760
81387 -41656
24198 17528
44918 -69665
50538 -56249
60992 22090
12465 -98979
22486 9745
86969 -34958
72451 -26928
41645 -63028
50922 -64018
82934 -89804
47552 6804
63282 -92529
34627 76656
16271 62482
2902 40314
8946 -13689
7288 48446
32723 3561
14634 -71001
91582 31429
74016 8713
42375 38176
83892 50006
98506 25687
43104 -45489
35047 12620
84605 97595
57252 29844
78003 30238
53262 16647
97547 -770
65848 -16240
88180 -90928
93463 -50766
31989 32928
5370 47368
3160 83089
92081 -27821
29216 -78509
11330 22371
65567 33671
35999 13401
40793 -48970
31886 -94640
86163 87486
2922 -26664
23363 -79868
34101 2812
71337 25006
83528 13689
40742 -87227
7715 -26143
57193 -94788
86712 99909
58
28550 82742
64084 28968
45205 -85989
42341 83730
14863 -88295
84821 -25227
71073 -58909
13356 -29420
47122 -50239
59907 -34623
8623 22100
46189 -81789
27810 -42233
64429 -23020
7898 18516
74519 -26415
70688 79285
42740 70847
80994 -40769
17183 52401
65455 55545
36207 5124
54493 -4355
52774 -68298
68644 97399
71603 -88015
73917 -10825
60893 9495
44192 67488
26236 7760
83936 58744
75779 -88150
44392 -58544
98569 -42908
22746 32746
10223 99529
60867 -78616
15690 -12177
53405 1506
30437 9741
41090 -27190
89237 -49685
87617 -29564
5018 80791
44539 -18856
13810 6967
43060 -8676
11753 17147
58290 -49869
54273 84419
68713 83218
36167 13528
8296 -13529
39562 35519
58356 -81200
37561 -47738
20385 2885
45896 55376
23
64199 6476
55647 16993
65276 -88263
69393 -27366
32449 18805
4092 96612
59469 -2762
94885 32336
8554 68559
94399 25037
79809 -12411
37826 23033
76299 -2581
54253 -53204
75817 12967
88603 35927
98140 -6537
18489 -61963
99938 -83391
83228 2265
73999 -96167
43689 -90873
58551 -17257
62
63022 37478
29640 1228
91623 24957
88077 -93539
16082 17921
85346 96327
90625 -24910
88588 -3023
6835 -33991
12255 -19928
58286 -30888
50886 55446
85336 -4868
31741 25887
27944 62811
70562 -81198
40808 -79451
75641 -7991
29019 87293
42184 37012
7236 48876
30414 66592
25947 8868
17831 10731
96139 6787
9378 178
53180 41079
79892 -80134
49825 -90682
98910 4882
5127 21719
29759 24724
21062 13547
46427 62387
83850 -52251
52954 57438
4642 25057
23199 23128
99835 -83714
58066 43711
26161 -70969
7709 -91652
65817 85142
68222 -76928
60926 93930
59511 -43469
73055 -3114
2925 -88519
30635 30215
44560 1655
73962 -98663
53075 60187
40695 -4377
62497 30459
35358 -88766
86443 -18221
87742 -85075
9184 96553
36811 -16942
84640 -19265
20560 -92388
41504 -89854
64
29663 94479
75450 -93572
71174 -98081
88351 -73905
4182 -42373
10137 -24024
89312 -45222
75071 56740
50162 -33142
37973 -70684
7113 -46555
76329 -46590
63463 97528
80576 -79396
54973 -43290
28962 -20201
2238 23665
79100 -15195
48955 -29936
34356 -66774
52828 -23041
12509 -5986
87929 -71060
53431 -42275
40988 -74771
6864 2678
71143 13937
39754 43403
93889 95429
36907 -41362
24818 -227
33986 27248
12101 -31016
8894 32240
21094 66958
35469 74242
64883 -71825
93942 2401
13461 -83170
34750 -97326
63019 35017
25956 86006
26236 -30473
4350 -91191
2339 -2965
88306 4290
91529 -25797
12183 53152
53301 58493
4596 -46200
39211 -38889
47711 -24416
7297 -67472
39889 -44623
90533 -66420
7297 -28071
36028 -45166
91776 -86116
29337 -61915
2573 46224
88198 -42772
10904 97636
46991 90754
72423 -13333
90
19649 72734
78103 -59849
27304 -37570
86933 80284
64178 38770
53062 84429
25810 30506
21973 87468
61218 -78117
16136 98198
99703 65590
82326 44418
46008 70084
36048 53106
65071 56033
83402 -13846
16038 95035
15523 48359
25683 -18746
55531 64350
65914 72924
85975 -56803
62309 -27775
7886 7931
85340 -69694
99823 -57925
34596 -42019
4481 28284
59974 89483
51047 37190
6218 81873
90266 22898
26712 -86462
38926 9747
52503 -959
11850 -38663
89167 -59231
34365 -94614
69516 84285
22311 -28739
5839 -7186
45603 31357
82930 2448
92681 -71286
48523 -61601
5484 75611
10372 36083
83913 -28477
41108 12415
18525 -2481
9727 -60554
53766 -88294
27322 15295
85309 -89300
95255 78977
28583 -34401
51920 -25615
56067 32669
70947 -1327
26047 -51860
59680 31970
47162 -21835
90950 37450
24984 59396
57673 17719
60951 -31493
91042 -51242
16180 92466
73492 66325
49736 -90093
74339 44376
30640 97956
67662 49563
35374 30953
61294 5125
18802 16760
37982 16318
72314 68219
25559 -35388
74514 27527
8227 -83606
92629 25323
85346 55760
25478 -32767
64925 -6949
60584 -80478
68778 -75663
2309 -23695
10738 -94947
64783 -32317
