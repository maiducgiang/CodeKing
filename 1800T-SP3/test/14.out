1
10
10
7
6
5
13
8
2
5
13
13
11
6
15
10
2
